workflow/count/src/main/java/com/datastax/oss/dsbulk/workflow/count/CountWorkflow.java

```diff
@@ -207,10 +207,12 @@ public void close() throws Exception {
       if (metricsManager != null) {
         metricsManager.reportFinalMetrics();
       }
-      // only print totals if the operation was successful
-      if (success) {
-        readResultCounter.reportTotals();
-      }
+
+      // Print results even if any failures.
+      // It has no sense to query billions rows for a few hours and show nothing if some failure
+      // happens
+      readResultCounter.reportTotals();
```

Comment:
That's much easier to review :-)

I understand the motivation but I am concerned that it is not possible to clearly tell whether the results are accurate or partial, because the operation failed.

How about we print a big fat warning before, if `success` is false?

Reply:
The big fat warning sounds good to me. As my experience with dsbulk, I always saw the success/failure status by the end of the log and that provides enough information to understand the quality of the results.
By the way, the count operation is always "approximate" when you running on the live cluster.
I'm not sure about message content, please, add any message you need.

Reply:
True, but still I think a little warning can't do any harm.

How about:

```
      readResultCounter.reportTotals();
      if (!success) {
        LOGGER.warn(
            "Please note: the totals reported above are probably inaccurate, "
                + "since the operation completed with errors.");
      }
```

Reply:
Looks awesome! Feel free to commit. You are the author :)

Reply:
Are you sure you don't want your name engraved for posterity in dsbulk's commit history? :-) I don't mind if you commit my suggestion as is, as long as you've signed the CLA.

Reply:
Thank you for the suggestion. The note was added as you proposed.